torch/fx/experimental/symbolic_shapes.py

```diff
@@ -295,6 +314,14 @@ def eval_guards(gm, *args):
 def bind_symbols(gm, *args):
     return gm.shape_env.bind_symbols(fx_placeholder_vals(gm), args)
 
+class DIM_DYNAMISM_STATE(Enum):
```

Comment:
nit: Conventionally, enum name is camel case, and the enum quantities themselves are all caps.